editor/docks/scene_tree_dock.cpp

```diff
@@ -4632,6 +4655,7 @@ void SceneTreeDock::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("_reparent_nodes_to_paths_with_transform_and_name"), &SceneTreeDock::_reparent_nodes_to_paths_with_transform_and_name);
 
 	ClassDB::bind_method(D_METHOD("_update_script_button"), &SceneTreeDock::_update_script_button);
```

Comment:
This can be removed now.

Reply:
Isn't there a chance existing code in games and plugins use this method since it seems to be exposed to the public API (e.g. via GDScript)?

Reply:
Methods starting with `_` are not part of public API.